### DISCUSSION TITLE
login  + problem with stair :((

### DISCUSSION THREAD

**Discussion by WhyNot-NotWhy** [IceHacks/SurvivCheatInjector#778] [General]
I can't login... Is there anyway to do that?
And maybe make it avalable to auto aim and shoot enimies at stairs?

(Anyone like me?)

**Comment by IceHacks** [comment]
I've had no problems logging in. I will check again tonight. What provider are you trying to use (Google, Twitch, etc.). As for the stairs, I have had some inspiration on how to make that possible. Surviv works in two different layers, ground and basement, as soon as you touch a stair, you are in the basement. This is plain 0 or 1, there is no blend. The AimBot doesn't aim at people below or above it, for obvious reasons. Checking if the enemy player is near a stair could solve this. Also, adding a toggle that makes the AimBot aim at underground people, but not shoot, has been considered.
